cg_hermes/cli/export.py

```diff
@@ -88,20 +89,15 @@ def export_tags_cmd(
     if not workflow:
         header = ["Tag name", "Description"]
         for category in COMMON_TAG_CATEGORIES:
-            table_name = category.upper().replace("_", " ")
+            table_name: str = " ".join(
+                re.findall(r"[A-Z](?:[a-z]+|[A-Z]*(?=[A-Z]|$))", category.__name__)
```

Comment:
This reg exp looks complex, is there any other way of structuring the data to avoid this one? Also, can one use category.name instead of `__name__`?

Reply:
Great suggestion, thanks! I made a class method :)